wsgi/config.py

```diff
@@ -0,0 +1,11 @@
+# see http://docs.gunicorn.org/en/latest/configure.html#configuration-file
+
+from os import getenv
```

Comment:
I switched to decouple and then for some strange reason the django app wouldn't read the settings from `.env`. Since this is a small file which I don't expect to change very often I just added `int` to cast the values

Reply:
That is really odd... No idea why that would be, but this works fine :)